src/python/WMCore/Services/DBS/ProdException.py

```diff
@@ -6,8 +6,7 @@
 
 """
 
-
-
+from future.utils import viewitems
```

Comment:
Just an observation, I could be wrong, but I think this code isn't used at all (perhaps since the migration to DBS3!)

Reply:
`Services/DBS/DBS3Reader.py` imports `from WMCore.Services.DBS.DBSErrors import DBSReaderError`. `DBSErrors.DBSReaderError` inherits from `DBSErrors.DataMgmtError`, which then inherits from `ProdException`

Reply:
Thanks for looking this up, Dario. I will stop making such unrelated observations ;)